Tests/Fixtures/Resources/YAML/good/mapping.yaml

```diff
@@ -4,3 +4,7 @@ foo:
   bar: banana
   baz: orange
 car: null
+names:
+  - John
+  - Peter
+  - Nick
```

Comment:
Not sure if this should be addressed by that PR or a separate one but I just realised that we might need to add test cases for strings containing (escaped) quotes like `- "Craig \"Hair Force One\" Federighi"` to see how our templates handle those… 

e.g. maybe we should at least use `##"raw string literals"##` in our stencil templates to limit the risk with those edge cases?

Reply:
Check #551, where we have issues with multiline strings. Whatever fix there should also escape quotes.

But yeah, that's for a separate PR.

Reply:
Ah cool, as long as that's tracked. Then let's merge this one and tackle the quote escapes and multilines issues separately via #551 👍